src/effects/native/echoeffect.h

```diff
@@ -26,14 +26,14 @@ struct EchoGroupState {
             : delay_buf(mixxx::AudioSignal::kSamplingRateMax * kMaxDelaySeconds *
                         kChannelCount) {
```

Comment:
With kMaxDelaySeconds as 2, this will not be big enough when the Time parameter is at its maximum and the BPM is below 60. Should kMaxDelaySeconds be increased? Is that edge case worth the higher memory consumption? These buffers consume a lot of memory, especially with #1254.